source/auth/2.0/index.md

```diff
@@ -65,29 +65,33 @@ Providing interoperable access to restricted content through client applications
 * Institutions have different existing access control systems.
 * Most IIIF viewers are client-side JavaScript applications, and may be served from a domain that is different from, and thus untrusted by, the image services that it is required to load.
 * Similarly, the domain of the authentication services may be different from that of a viewer or the IIIF-based content. Therefore, the authorizing server must not require any prior knowledge of the domain hosting the viewer.
+<!-- 
+How true is this? ^^^ The domain of the token service must be the same as the content!
+The access service _could_ be different, but would need to visit the content domain at some point to set a cookie.
+ -->
 
 Additionally, the IIIF community has the following goals for this specification:
 
-* A IIIF client should not authenticate the user itself; the server hosting the content must be responsible for capturing credentials from a user and the IIIF viewer needs no knowledge of or access to this exchange.
-* A browser-based IIIF client must be able to maintain its own internal state during an authentication flow.
+* A IIIF client should not accept credentials and authenticate the user itself; the server hosting the content must be responsible for capturing credentials from a user and the IIIF viewer needs no knowledge of or access to this exchange.
+* A browser-based IIIF client must be able to maintain its own internal state during an authentication flow. That is, it must be able to stay running while the user interacts with third parties in another tab.
 * A registry of trusted domains should not be required; anyone should be able to create any kind of viewer and run it from anywhere.
-* Institutions should be able to use their existing authentication systems without modification.
+* Institutions should be able to use their existing authentication systems without modification: this specification can provide a bridge to existing systems without requiring that the systems themselves be changed.
```

Comment:
Institutions should be able to work with existing authentication systems without modification?